README.md

```diff
@@ -0,0 +1,54 @@
+# statick-action
+
+![Self Test](https://github.com/gregtkogut/statick-action/workflows/Self-Test/badge.svg?branch=action-experimentation)
+
+This is a **GitHub Action** that runs the **[statick](https://github.com/sscpac/statick)** static code analysis tool.
+
+The action inputs mirror the command line arguments for the CLI version of
+statick, and much more detail about the powerful statick configuration options is
+available in the README of the above repository.
+
+## Inputs
+
+## `check`
+
+Any discovered statick issue fails the action. Default `true`.
```

Comment:
```suggestion
Any discovered Statick issue fails the action. Default `true`.
```

Reply:
Right now the `--check` parameter in Statick defaults to `False`. We do this so that the `statick` command can be piped together with other Unix commands and will not stop a pipeline just because issues are discovered.